readme.md

```diff
@@ -352,6 +352,29 @@ Create a new `ky` instance with some defaults overridden with your own.
 
 In contrast to `ky.create()`, `ky.extend()` inherits defaults from its parent.
 
+You can remove a header with extend() by passing headers: `{[key:string]: undefined}`
+You can pass a Headers instance or a plain object
+
+```js
+import ky from 'ky';
+
+const original = ky.create({
+	headers: {
+		rainbow: 'rainbow',
+		unicorn: 'unicorn'
+	}
+});
+
+const extended = original.extend({
+	headers: {
+		rainbow: undefined
+	}
+});
+
+console.log('rainbow' in extended); // false
+console.log('unicorn' in extended); // true
```

Comment:
This doesn't seem right. `extended.unicorn` is `true`?

Reply:
I've updated the PR. I meant `response` instead of extended.